pkg/openshift/rosa/oidcconfig.go

```diff
@@ -45,11 +46,16 @@ func (r *Provider) createOIDCConfig(ctx context.Context, prefix, installerRoleAr
 		"--output", "json",
 		"--mode", "auto",
 		"--prefix", prefix,
-		"--managed=false",
```

Comment:
This defaults to true, if you remove it as false, will this not break the commercial flow?

```
      --managed           Indicates whether it is a Red Hat managed or unmanaged (Customer hosted) OIDC Configuration. (default true)
```

Reply:
I think this still needs updating unless I'm missing it